internal/services/dashboard/dashboard_grafana_managed_private_endpoint_resource.go

```diff
@@ -105,6 +106,12 @@ func (r ManagedPrivateEndpointResource) Arguments() map[string]*pluginsdk.Schema
 			Optional:     true,
 			ValidateFunc: validation.StringIsNotEmpty,
 		},
+
+		"private_link_service_url": {
```

Comment:
should we also set this property in the update func and cover this in the testing too?

Reply:
done